x-pack/filebeat/module/google_workspace/admin/config/config.yml

```diff
@@ -15,7 +15,7 @@ request.transforms:
   - set:
       target: url.params.startTime
       value: "[[.cursor.last_execution_datetime]]"
-      default: '[[parseDate now (parseDuration "-{{.initial_interval}}")]]'
+      default: '[[formatDate (now (parseDuration "-{{.initial_interval}}"))]]'
```

Comment:
It would be nice to add some test cases, but I understand it may be hard to do this in Beats.